test/coverage-system/renderer.test.ts

```diff
@@ -1,20 +1,28 @@
-import {assert} from "chai";
-import {Section} from "lcov-parse";
-import {DecorationOptions, Range, TextEditor, TextEditorDecorationType} from "vscode";
-import {Renderer} from "../../src/coverage-system/renderer";
-import {SectionFinder} from "../../src/coverage-system/sectionfinder";
+import { assert } from "chai";
+import { Section } from "lcov-parse";
+import { DecorationOptions, Range, TextEditor, TextEditorDecorationType } from "vscode";
+import { Renderer } from "../../src/coverage-system/renderer";
+import { SectionFinder } from "../../src/coverage-system/sectionfinder";
+import { StatusBarToggler } from "../../src/extension/statusbartoggler";
 import { fakeConfig } from "../mocks/fakeConfig";
 
 suite("Renderer Tests", function() {
+    let statusBar: StatusBarToggler;
+
+    beforeEach(() => {
+        statusBar = new StatusBarToggler(fakeConfig);
+    });
```

Comment:
sadly I haven't updated to the latest testing bits for extensions so you need to use the super old naming style for this (setup I believe) #263 . I will fix this for yah though 🤓 .

```typescript
    setup(function() {
        (vscode as any).window.createTextEditorDecorationType = fakeVscode.createTextEditorDecorationType;
        (vscode as any).commands.executeCommand = fakeVscode.executeCommand;
        (vscode as any).workspace.getConfiguration = fakeVscode.getConfiguration;
    });
```

Reply:
I've implanted #263 locally and will be publishing a PR shortly 👍